circle.yml

```diff
@@ -26,7 +26,29 @@ jobs:
       - store_test_results:
           path: .
 
-  build-frontend:
+  frontend-quality:
+    docker:
+      - image: circleci/node:lts
+    working_directory: ~/project/frontend
+    steps:
+      - checkout:
+          path: ~/project
+      - restore_cache:
+          key: dependencies-{{ checksum "package-lock.json" }}
```

Comment:
@Agrendalath I tried it, and it doesn't work. Following the documentation instructions the CI throws an error:
```
# |   |   2. [#/jobs/frontend-build/steps/1/restore_cache] only 1 subschema matches out of 2
# |   |   |   1. [#/jobs/frontend-build/steps/1/restore_cache/key] expected type: String, found: Sequence
# |   |   |   |   SCHEMA:
# |   |   |   |     type: string
# |   |   |   |   INPUT:
# |   |   |   |     null
```
And the error happens for cache restores that are using a list of arguments instead of a string. :disappointed: 
Do we have any repo with this working so that I can check/copy?

Reply:
See [this build](https://circleci.com/gh/open-craft/opencraft/9659?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

Reply:
I checked your commits and it looks like you were using `key` instead of `keys` for restoring multiple cache options ;)

I pushed the commit to the separate branch, [verified it on the CI](https://circleci.com/workflow-run/dc915f85-1bbf-4806-8baf-d3256f1426c2) and cherry-picked it to this branch.

Reply:
Woah, thanks for fixing that for me! I didn't even notice that there was a difference in the usages.